app/store/mutations.js

```diff
@@ -57,16 +58,16 @@ export const FIREBASE_INIT = (state, store) => {
             okButtonText: 'OK'
           }).then(() => {
             console.log('Alert dialog closed')
-            store.commit('SET_FOREGROUND', true)
+            store.commit(ActionTypes.SET_FOREGROUN, true)
```

Comment:
I think we missed a ***D*** in foreground.

Reply:
It was a carelessness. I go to correct it